setup.py

```diff
@@ -766,9 +766,9 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'jdbc': ["jdbc"],
     'jenkins': ["jenkins"],
     'jira': ["jira"],
-    'kerberos': [],
+    'kerberos': ["kerberos"],
```

Comment:
Jarek, looks like `apache.beam` one is also empty (Line 718) -- should it be empty, it should have `apache.beam` right?

Similarly I am not sure if the following should be empty too:

- dask
- apache.atlas
- rabbitmq
- sentry
- tableau

WDYT ?

Reply:
I will review it fully,

Reply:
actually I remove back statsd/kerberos. We do not have neither of them as providers :)

Similarly there are no providers for dask, apache.atlas, rabbitmq, sentry. There is one for rableau but it correctly has no additional requirements. 

So just correcting LDAP was right intiially. I removed stasd/kerberos.

Reply:
In our case extras <> providers. We have  quite a few more extras than providers . We need to refactor setup.py badly  to reflect that because it is awfully confusing (includig myself as you see) . We already discussed it in #12548 and I am going to do this very thing before New Year. I created #13309 to not forget about it.

Reply:
🤦 🤦 . There is also no `ldap` provider :(. This is a different issue altogether

Reply:
Seems that we simply miss `python-ldap` dependency (and it goes as far as 1.10.4 https://issues.apache.org/jira/browse/AIRFLOW-5261 and  https://apache-airflow.slack.com/archives/C0146STM600/p1608836687492700